src/main/java/org/alfresco/rest/core/JsonBodyGenerator.java

```diff
@@ -1,13 +1,14 @@
 package org.alfresco.rest.core;
 
-import java.util.HashMap;
```

Comment:
Minor: missing lic headers (I realise not specific to this PR) ?

Reply:
After a quick survey of the project then I didn't spot any classes with license headers except the FolderRules class created by Kris.  I'll work on a second PR to add the license header plugin to this project.